starlark/profile.go

```diff
@@ -345,9 +346,7 @@ func profile(w io.Writer) {
 		enc.bytes(Profile_sample, sample.Bytes())
 	}
 
-	end := nanotime()
-	enc.int(Profile_time_nanos, start)         // start time of profile
-	enc.int(Profile_duration_nanos, end-start) // duration of profile
+	enc.int(Profile_duration_nanos, time.Since(start).Nanoseconds())
```

Comment:
This fixes #186, right?

If the start time is the only issue, I think it makes sense to record the start time with `time.Now`, but call `nanotime` twice separately to measure the duration.

Reply:
Good point. Done.